ep/build-velox/src/get_velox.sh

```diff
@@ -107,7 +107,7 @@ function process_setup_centos9 {
   sed -i '/^.*dnf_install autoconf/a\  dnf_install libxml2-devel libgsasl-devel libuuid-devel' scripts/setup-centos9.sh
   
   ensure_pattern_matched 'install_gflags' scripts/setup-centos9.sh
-  sed -i '/^function install_gflags.*/i function install_openssl {\n  wget_and_untar https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1s.tar.gz openssl \n  cd openssl \n  ./config no-shared && make depend && make && sudo make install \n  cd ..\n}\n'     scripts/setup-centos9.sh
+  sed -i '/^function install_gflags.*/i function install_openssl {\n  wget_and_untar https://github.com/openssl/openssl/releases/download/openssl-3.2.2/openssl-3.2.2.tar.gz openssl \n  cd ${DEPENDENCY_DIR}/openssl \n  ./config no-shared && make depend && make && sudo make install \n}\n'     scripts/setup-centos9.sh
```

Comment:
You have to use curly braces if you want to cd in a subshell and avoid `cd ../..` `( cd ${DEPENDENCY_DIR}/openssl \n  ./config no-shared && make depend && make && sudo make install )`

Reply:
updated. Will keep this in mind

Reply:
@majetideepak please review and lmk if any more changes are required. Thanks.